config/locales/activerecord.yml

```diff
@@ -247,6 +248,9 @@ en:
         publishers/job_listing/job_details_form:
           attributes:
             <<: *job_details_errors
+        publishers/job_listing/working_patterns_form:
+          attributes:
+            <<: *working_patterns_errors
```

Comment:
Needs to be added around line 500 too. That's why this is assigned to a variable using the `&`: so it can be reused without duplication.